examples/prompting/Basic_Classification.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>As in the other PR, don't use "Gemini" by itself unless you're talking about the chatbot.</p><p> </p><p>Also nit - "off-topic" has no plural.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>